### DISCUSSION TITLE
Starting off with GSOC 2025 Contribution

### DISCUSSION THREAD

**Discussion by Rishi2003Das** [camicroscope/GSOC#68] [General]
Hello everyone! I am *Rishi Das* from National Institute Of Technology, Rourkela. I am willing to participate as a *contributor in GSOC 2025* and while checking out the past projects I find your organization very interesting and relatable to my work. I am an undergrad engineering student studying in my 3rd year of graduation. I am very enthusiastic about working in the fields of AI/ML and I am also doing *research in this field of medical imaging. I mainly work on Computer Vision projects related to **Medical Imaging like Cancer detection and classification*. I am interested to contribute in your projects like Image Classification  using Foundation Models. Please mention what are the Future Project Plans of caMicroscope for aspiring GSOC Contributors of 2025 so, that I can start contributing to your project. I can work on most issues and features rated to Imaging preprocessing, augmentation, segmentation and classification, so @birm , you can assign me any task or issues that you want me to contribute to. Further, I also have a few feature modifications and improvements in my mind so, if you guide me on where to propose them I will surely approach them soon. I am happily open to all!
Thanking you,
Rishi Das.
[LinkedIn](https://www.linkedin.com/in/rishi-das-522537250/)
[Kaggle](https://www.kaggle.com/rishi2003das)
[Email](rishikakalidas@gmail.com)

**Comment by vivek6337** [comment]
Hello everyone !
I am **Vivek Kumar** , a 2nd yr chemical engg. undergrad student at **IIT BHU** . I am experienced in Computer Vision , NLP and also classical ML . Also i have done various projects related to application of ML in medical sciences like  **visual question - answering on radiology images , disease detection classification & segmentation** etc.. I am very much interested in this field and would like to contribute to caMicroscope for GSOC 2025 . @birm can you give me any task in which i can contribute

**Comment by highpinginfinity** [comment]
HELLO EVERYONE!
I am Satyam Sinha from IIT BHU, Varanasi, India. I am highly interested in ML. I have strong grasp on topics of DL and have keen interest in LLMs. I have good hold on CV and also I am currently exploring NLP. My skillset includes PyTorch, TensorFlow, CNNs and RNNs. I am highly enthusiastic to apply ML and CV in medical field. I have done some projects in this area such as Cancel Detection and Automating Counting Cells. I am eager to learn from the community and contribute here. Can someone please guide me through it?
I will be happy to contribute to the community. 
Thank You

**Comment by akashlogics** [comment]
Dear caMicroscope Selection Committee,

I hope this message finds you well. I am writing to express my keen interest in participating in the Google Summer of Code (GSOC25) program and contributing specifically to the caMicroscope project.

I am currently pursuing a degree in Artificial Intelligence and Data Science, where I have built a strong foundation in programming, data analysis, and machine learning. My academic journey has allowed me to explore and develop proficiency in tools and technologies that align closely with the goals of caMicroscope, such as Python programming, data visualization, and large-scale image processing.

In addition to my coursework, I have actively engaged in practical projects that bridge theory and application. For instance, I have worked with libraries like pandas, NumPy, and OpenCV to process and analyze complex datasets, including medical imaging data. I have also implemented algorithms in Python and C++ for efficient image segmentation and pattern recognition tasks.

These experiences have equipped me with a solid understanding of digital pathology workflows and an appreciation for the challenges and opportunities in building software for high-resolution image analysis. caMicroscope’s focus on enabling collaborative exploration of pathology data resonates deeply with my interests and aspirations.

I am eager to apply my technical skills, analytical mindset, and enthusiasm for computer vision and data science to contribute meaningfully to the caMicroscope project. Joining your team would be an incredible opportunity to support advancements in medical research and technology while growing alongside a community of like-minded contributors.

Thank you for considering my application. I am excited about the potential to work with the caMicroscope community and contribute to impactful, open-source innovation.

**Comment by skartik04** [comment]
I am Kartik, a first-year PhD student in Electrical and Computer Engineering at UCLA. My current research focuses on analyzing transformer circuits used in large language models (LLMs). Additionally, I am working on utilizing vision-language models (VLMs) to analyze epileptic seizures by processing real patient videos. For this project, I am employing the Qwen-7B model to identify and classify different bodily movements during seizures into two major categories: epileptic and non-epileptic seizures. I believe this work is similar to the research conducted in your field, which means I have developed some relevant expertise and would be eager to contribute to your group’s efforts.

Furthermore, I am deeply interested in expanding my knowledge and expertise in biology-related machine learning tasks and LLM applications. I am enthusiastic about contributing to and learning from the cutting-edge research conducted within this group.

**Comment by anishshrma** [comment]
Hello everyone!

I am Anish Sharma, a 3rd-year undergraduate student at Model Institute of Engineering and Technology, Jammu. I am excited to participate as a contributor in GSoC 2025, and after exploring past projects, I found caMicroscope very relevant to my interests and work.

I have a keen interest in AI/ML, particularly in Medical Imaging and Cancer Detection. I am currently working on a cancer detection website, where users can upload images to analyze the probability of cancer. My primary skills include image preprocessing, augmentation, segmentation, and classification.

I am eager to contribute to projects like Image Classification using Foundation Models and would love to know more about the future project plans of caMicroscope for GSoC 2025 contributors. If there are any open tasks or issues related to imaging, @birm, feel free to assign them to me!

Additionally, I have some feature improvement ideas in mind—please guide me on where to propose them, and I will approach them accordingly. Looking forward to learning and contributing!

Thank you!
Anish Sharma
📧 Email: anishkhour@gmail.com
🔗 GitHub: [github.com/anishshrma](https://github.com/anishshrma)
💼 LinkedIn: [linkedin.com/in/anish-sharma-87601a235](https://www.linkedin.com/in/anish-sharma-87601a235/)

**Comment by its-pratyushpandey** [comment]
Hi Rishi,
Thanks for your interest in caMicroscope and for sharing your background — your work in medical imaging and computer vision aligns well with the goals of the project. For GSoC 2025, we’re currently reviewing potential ideas and would love input from contributors like you.

You can start by exploring open issues in the repository and contributing to areas like preprocessing, segmentation, or model integration. Feel free to share your improvement suggestions by opening a GitHub issue or discussing them on our mailing list or Slack.

Looking forward to your contributions!

Best,
[Pratyush Kumar Pandey]

**Reply by birm** [reply]
We were not selected for GSOC 2025 as an organization, so we are not participating this year.

**Comment by Rishi2003Das** [comment]
Hello @its-pratyushpandey ,
Thanks for the reply. I am quite interested to give new ideas and contribute to your projects. I will be contributing and raise issues wherever required or deemed necessary. Thank you for the reply once again.
Best Regards,
Rishi Das.
